index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <link rel="icon" href="images/header-background-mobile.png" type="image/x-icon"/>
+    <title>Supreeti Portfolio</title>
+  </head>
+  <body>
+    <header class="header1">
+        <h3 class="logo"><a href="#">My Logo</a></h3>
+          <nav>
+              <img src="images/Union.png" alt=""/>
+          </nav>
+    </header>
+    <main>
+        <section class="section">
+            <h1 class="title">I’m Supreeti Glad to see you!</h1>
```

Comment:
- [ ] [ OPTIONAL ] You can consider using a line break (`br` tag) between your name and "Glad", to make it easier to style this heading.